tests/system/test_sqlalchemy_bigquery.py

```diff
@@ -34,6 +34,7 @@
 import datetime
 import decimal
 
+sqlalchemy_version_info = tuple(map(int, sqlalchemy.__version__.split(".")))
```

Comment:
This works, though I've been trying to use https://packaging.pypa.io/en/latest/version.html which is the canonical version parser. We already have it pulled in via a transitive dependency through `setuptools`, I believe.

Reply:
Nice. I'll switch to that.

WRT how we get it, IMO, anything we import should be a direct dependency.

Reply:
Sounds good. I'm fine explicitly including it in our dependencies / test dependencies.

Reply:
Done